src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+function avarageCalculator(ages, peopleCount) {
```

Comment:
Why do you need to pass the array's length as an argument?
You could calculate it in the function body